core/drivers/crypto/hisilicon/include/hisi_qm.h

```diff
@@ -147,6 +147,13 @@ struct hisi_qp {
 	enum hisi_drv_status (*parse_sqe)(void *sqe, void *msg);
 };
 
+struct qm_rsv_buf {
```

Comment:
This content of this struct is identical to the
``` C
	struct qm_sqc *sqc;
	struct qm_cqc *cqc;
	paddr_t sqc_dma;
	paddr_t cqc_dma;
```
in `struct hisi_qm`. How about renaming `struct qm_rsv_buf` to something neutral, like `struct qm_buf` (or `struct qm_xqc` or even `struct qm_qc`)  and then use it twice in `struct hisi_qm`?

What does `rsv` stand for? It seems that set of buffers only are used from `qm_sqc_cfg()` and `qm_cqc_cfg(). What makes those two functions special?

Reply:
Yes.This only used from qm_sqc_cfg() and `qm_cqc_cfg(). qm_rsv_xqc is used when writing mailbox to config xqc. Without this patch ,we do memalign when we need to use it.But it caused a problem (When the mailbox operation times out, the software will free the temporary memory. The hardware does not cancel the mailbox operation and may continue to read and write the free memory).

Reply:
I will rename it to qm_xqc, but I prefer to make the smallest modifications to struct hisi_qm.

Reply:
Is this OK?
struct hisi_qm {
	struct qm_sqc *sqc;
        struct qm_cqc *cqc;
	paddr_t sqc_dma;
	paddr_t cqc_dma;
	struct qm_xqc xqc;

Reply:
My point with renaming `struct qm_rsv_buf` was to be able to reuse it, if you're not going to reuse it then it doesn't matter.